client/landing/gutenboarding/components/titles.scss

```diff
@@ -2,17 +2,22 @@
 @import '../mixins';
 
 .gutenboarding-title {
+	@include onboarding-heading-text-mobile;
 	color: var( --mainColor );
-}
-.gutenboarding-title {
-	@include onboarding-font-recoleta;
-	font-size: 42px;
+
+	@include break-mobile {
+		@include onboarding-heading-text;
+	}
 }
 
 .gutenboarding-subtitle {
-	font-size: 14px;
+	@include onboarding-large-text;
```

Comment:
In the [design](https://hardcore-dubinsky-43a2f5.netlify.com/), the subtitle font size is 14px (`onboarding-medium-text`).

Reply:
I guess the existing implementation was based on Figma. @dubielzyk if you think this needs to be changed, please feel free to add to https://github.com/Automattic/wp-calypso/issues/40757